compute/instances/reference-content/cost-optimized.mdx

```diff
@@ -0,0 +1,75 @@
+---
+meta:
+  title: The right Instance for development purposes
+  description: Find out more about Instances from Scaleway's Cost-Optimized range, such as Development and General Purpose Instances.
+content:
+  h1: The right Instance for development purposes
+  paragraph: Find out more about Instances from Scaleway's Cost-Optimized range, such as Development and General Purpose Instances.
+dates:
+  validation: 2023-02-22
+  posted: 2023-02-22
+categories: 
+  - compute
+tags: instance, type, development, DEV, range, cost-optimized, general-purpose, general, purpose, GP, PLAY2, PRO2, play, pro
```

Comment:
I think we don't use commas in the tags section ?

Reply:
you're right je vais fix